.github/workflows/cmake.yml

```diff
@@ -48,7 +53,13 @@ jobs:
 
     - name: make re/rem
       run: |
-        for p in re rem; do cd $p && sudo PATH="$PATH" make install && cd - && sudo rm -Rf $p; done
+        for p in re rem; do
+          cmake -S $p -B $p/build
+          cmake --build $p/build
+        done
+        for p in re rem; do
+          mv $p ../.
+        done
```

Comment:
this one changes building of re/rem from make to cmake

the last step is a clever trick to move re/rem one level up, so that baresip can find them

please review if this is a good idea (please review this hunk)